src/state/token_index.ml

```diff
@@ -0,0 +1,83 @@
+open Yojson
+open Lwt
+open Learnocaml_data
+
+
+let token_file = "sync/token.json"
+
+(* Unlocked *)
+let mutex_token = Lwt_mutex.create ()
+
+let cast_list l = `List l
+
+let string_to_json (value:string) = (`String value : Yojson.Basic.t)
+
+let token_to_string l = List.map (fun t -> Token.to_string t) l
```

Comment:
```suggestion
```

this function seems unused, and basically amounts to `List.map Token.to_string` so maybe we can remove it?